src/orderdef.h

```diff
@@ -89,6 +89,7 @@ typedef DroidOrderType DROID_ORDER;
 enum SECONDARY_ORDER
 {
 	DSO_UNUSED,
+	DSO_ATTACK_RANGE,               /**< The attack range a given droid is allowed to fire: can be short, long or default (best chance to hit). */
```

Comment:
Since "Long Range" is the default setting, the comment is misleading.

Reply:
Fixed.